packages/release/activate-configured.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=Isolates configured.target
+After=preconfigured.target
+Requires=preconfigured.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl isolate configured
```

For consistency, should we also set the default?
```suggestion
ExecStart=/usr/bin/systemctl set-default configured
ExecStart=/usr/bin/systemctl isolate configured
```

---

I could but I wanted to minimize the calls to `set-default` because it will effectively create/remove symlinks.

---

I'd prefer the consistency and the ability to use command-line tools to inspect what state the host system thinks it's reached.